### PR TITLE
update notebook tests for new notebooks

### PR DESCRIPTION
**Pull request by imatiach-msft**
follow-up to PR:
https://github.com/microsoft/responsible-ai-toolbox/pull/1062
to fix the notebook tests to point to the new notebooks

### CODE REVIEW

File: notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb [hunk 1]
```diff
@@ -94,6 +95,17 @@
     "\n",
     "target_feature = 'SalePriceK'\n",
     "categorical_features = []\n",
+    "\n",
+    "outdirname = 'responsibleai.12.6.21'\n",
+    "try:\n",
+    "    from urllib import urlretrieve\n",
+    "except ImportError:\n",
+    "    from urllib.request import urlretrieve\n",
```
Comment (kicha0) [inline notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb:103]: just for background context, is this attempting to deal with different major versions of urllib with breaking api change?

**Review by kicha0 — APPROVED** [review]

File: notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb [hunk 2]
```diff
@@ -122,7 +122,7 @@
     "    print(pipe_cfg['cat_cols'])\n",
     "    return X, feat_pipe, features, classes\n",
     "\n",
-    "outdirname = 'erroranalysis.12.3.20'\n",
+    "outdirname = 'responsibleai.12.6.21'\n",
```
[the remark this answers is not in the document]
Comment (imatiach-msft) [reply notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb:125]: This is the zip file that gets uploaded.  I add a date to it so if any dataset is removed or I happen to break something like folder structure it won't affect old branches/tests/notebooks.

File: notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb (hunk 1 above)
Comment (imatiach-msft) [reply notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb:103]: yep, exactly, different versions of urllib

Comment (imatiach-msft) [reply notebooks/responsibleaidashboard/responsibleaidashboard-housing-decision-making.ipynb:103]: or actually, I think it was even more complicated than that. I recall installing pandas somehow even modifies this path.  I might be mis-remembering though.

**Pull request merged**